packages/abi-gen/src/utils.ts

```diff
@@ -92,4 +93,24 @@ export const utils = {
             inputs: [],
         };
     },
+    makeOutputFileName(name: string): string {
+        let fileName = toSnakeCase(name);
+        if (fileName === 'z_r_x_token') {
+            fileName = 'zrx_token';
+        }
+        return fileName;
+    },
+    writeOutputFile(filePath: string, renderedTsCode: string): void {
+        fs.writeFileSync(filePath, renderedTsCode);
+    },
+    isOutputFileUpToDate(abiFile: string, outputFile: string): boolean {
+        const abiTime = fs.statSync(abiFile).mtimeMs;
```

Comment:
The variable name should elucidate the units of time used. Sec? Ms? Hours? It is clear on this line, but not further down where the variable is reused. E.g `abiFileModifiedTimeMs`